src/__experimental__/components/fieldset/fieldset.component.js

```diff
@@ -23,11 +23,20 @@ const Fieldset = (props) => {
     return null;
   };
 
+  const legendInline = () => {
```

Comment:
I think we can simplify this by removing this function and using `props.legendInline` below.